src/movies.js

```diff
@@ -1,16 +1,117 @@
-// Iteration 1: All directors? - Get the array of all directors.
-// _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
+/*  Iteration 1: All directors? - Get the array of all directors.  */
+/*  Bonus: It seems some of the directors had directed multiple movies,
+    so they will pop up multiple minutess in the array of directors.
+    How could you "clean" a bit this array and make it unified (without duplicates)? */
+const getAllDirectors = (movies) => {
+  return movies.map((movie) => movie.director);
```

Comment:
simply amazing, one line solution! can't wait to see you solving katas like a pro >:)

Reply:
if you want to reduce the syntax even more you don't need to use parenthesis in the parameter when you only have one parameter.